workflows/genomes_annotation.nf

```diff
@@ -265,20 +276,30 @@ workflow GAP {
         MMSEQ_SWF.out.mmseq_90_cluster_rep_faa,
         all_prokka_fna,
         cluster_reps_gbks,
+        cluster_reps_faas,
+        cluster_reps_gffs,
         species_reps_names_list,
         ch_interproscan_db,
         ch_eggnog_db,
         ch_eggnog_diamond_db,
         ch_eggnog_data_dir,
-        ch_rfam_rrna_models
+        ch_defense_finder_db,
+        ch_dbcan_db,
+        ch_antismash_db
     )
-
+    
+    DETECT_RNA(
+        all_prokka_fna,
+        accessions_with_domains_ch,
+        ch_rfam_ncrna_models
+    )
+    
     METADATA_AND_PHYLOTREE(
         cluster_reps_fnas.map({ it[1]}).collect(),
         all_prokka_fna.map({ it[1] }).collect(),
         PREPARE_DATA.out.extra_weight_table,
         PREPARE_DATA.out.genomes_checkm,
-        ANNOTATE.out.rrna_outs,
+        DETECT_RNA.out.rrna_outs.flatMap {it -> it[1..-1]}.collect(),
```

Comment:
what is this sorcery `[1..-1]`?

Reply:
@mberacochea this takes everything from the tuple except for the element with index 0 (everything from element #1 through the last element). Would you do this differently?

Reply:
That's fine, Groovy